### ISSUE TITLE
Code analysis reporting bogus warning messages without a issue number

### ISSUE THREAD

**Issue by jmecosta** [dotnet/sdk#30592]
Hi,

We are referencing a netstandard2.0 from a C++ CLI (net 48). We hit https://github.com/dotnet/sdk/issues/1781 after supressing the warning: C4691 we get flooded with:

2>file.hpp(30,1): message : This diagnostic occurred while importing type 'TYPEONE ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(30,1): message : This diagnostic occurred while importing type 'TYPEONE ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(30,1): message : This diagnostic occurred while importing type 'TYPEONE ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(40,1): message : This diagnostic occurred while importing type 'TYPETWO ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(40,1): message : This diagnostic occurred while importing type 'TYPETWO ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(40,1): message : This diagnostic occurred while importing type 'TYPETWO ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(13,1): message : This diagnostic occurred while importing type 'TYPETWO ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)
2>file.hpp(13,1): message : This diagnostic occurred while importing type 'TYPETWO ' from assembly 'Tekla.Structures.ObjectDimensioningModel, Version=2024.0.0.0, Culture=neutral, PublicKeyToken=2f04dbe497b71114'. (compiling source icon32 file.cpp)

Basically this looks like a bug from the codeanlaysis since : message : is not a valid warning. It should contain a valid issue to supress

VS 2022  Version 17.4.4

Thanks in advanced
Jorge Costa

**Comment by jmecosta** [comment]
Which means it will never get addressed. Since Microsoft wants everyone to
stop using c++ cli.

Ok u may close this ticket. I open it because I saw somewhere code analysis
in the log so I tought it could be from the .net side.

**Comment by jmecosta** [comment]
Msbuild option. Pragma didn't work

**Comment by baronfel** [comment]
That's right! @jmecosta thanks for the report, but the team that could solve your error response to VS Feedback as @KalleOlaviNiemitalo noted. Can you reopen there? Thanks, and sorry for the confusion.

**Comment by jmecosta** [comment]
Nah it's pointless.  We will just get rid of the all c++ cli. Not much we
can do
